Bio_Files_Processor.py

```diff
@@ -0,0 +1,126 @@
+import os
+from typing import List
+
+def convert_multiline_fasta_to_oneline(input_fasta: str, output_fasta: str = None) -> str:
+    """
+    Function conver multiline fasta file into fasta with name line marked by '>' and the other line with sequence.
+    Results save into new folder "Converted_data". If the folder already exists, new data is written to it.
+    
+    :param input_fasta: Path to fasta-file with your seqs with. 
+     !! It is necessary to indicate the name along with extensions (.fasta) !!
+    :type input_fasta: str
+    :param output_fasta: Name of new fasta-file with your seqs in one line, default value = None
+    It is necessary to indicate the name along with extensions (.fasta)
+    :type output_fasta: str
+    :rtype: str
+    :return: Script completion message 
+    """
+    if input_fasta.find('.fasta') == 0:
+        raise ValueError(f'Wrong file format in input!')
+    if os.path.exists(os.path.join('.', 'Converted_data')) == False:
+        os.mkdir(os.path.join('.', 'Converted_data'))
+    if output_fasta == None:
+        out_name = 'one_line_' + input_fasta.strip("/")[-1]
```

Comment:
Я так понима, тут подразумевался split? 
В любом случае, решение хорошее, но мало ли будет другая ОС? Для этого есть функций `os.basename`, она тут вообще супер подходит
```suggestion
        out_name = 'one_line_' + os.basename(input_fasta)
```